packages/jest-haste-map/src/index.js

```diff
@@ -39,6 +39,7 @@ type Options = {
   cacheDirectory?: string,
   console?: Console,
   extensions: Array<string>,
+  forceNativeFind?: boolean,
```

Comment:
... and as I say this I just realised I should actually be forcing the other thing and I've been thinking about it backwards haha